scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java

```diff
@@ -1227,4 +1225,14 @@ public TaskState toTaskState() {
         return taskState;
     }
 
+    @Transient
+    public boolean isForkTask() {
+        if (taskType.equals(SCRIPT_TASK)) {
```

Comment:
if i am not totally sure that String is not null, then i do:
```suggestion
        if (SCRIPT_TASK.equals(taskType)) {
```

Reply:
Good point, just applied.